ui/src/components/chip/__tests__/QChip.spec.js

```diff
@@ -1,154 +1,467 @@
+import { mount } from '@cypress/vue'
+import QChip, { defaultSizes } from '../QChip.js'
+
+const defaultOptions = {
+  label: 'simple chip'
+}
+
+function mountQChip (options = {}) {
+  options.props = {
+    ...defaultOptions,
+    ...options.props
+  }
+
+  return mount(QChip, options)
+}
+
 describe('Chip API', () => {
   describe('Props', () => {
     describe('Category: content', () => {
       describe('(prop): icon', () => {
-        it.skip(' ', () => {
-          //
+        it('should render an icon', () => {
```

Comment:
This will pass even when using `icon-right`
I guess we should change this to `should render an icon on the left` and check for `q-chip__icon--left`